pkg/analytics/analytics.go

```diff
@@ -60,6 +60,7 @@ const (
 	previewDestroyEvent      = "DestroyPreview"
 	execEvent                = "Exec"
 	signupEvent              = "Signup"
+	setContextEvent          = "SetContext"
```

Comment:
```suggestion
	contextEvent          = "Context"
```
We are using the command name for other metrics